pkg/cache/cache.go

```diff
@@ -314,11 +328,13 @@ func (c *Cache) AddClusterQueue(ctx context.Context, cq *kueue.ClusterQueue) err
 	for _, q := range queues.Items {
 		qKey := queueKey(&q)
 		qImpl := &queue{
-			key:               qKey,
-			admittedWorkloads: 0,
-			usage:             make(FlavorResourceQuantities),
+			key:                qKey,
+			reservingWorkloads: 0,
+			admittedWorkloads:  0,
+			usage:              make(FlavorResourceQuantities),
```

Comment:
Are you keeping `usage` to mean the reserved quota?
If you are doing this to minimize the diff, that's ok, but please add a comment in the queue struct, with a TODO to rename in the future.